ml/src/main/scala/org/apache/spark/ml/spark/models/svm/SVMModel.scala

```diff
@@ -28,6 +31,7 @@ object SVMModel {
     var interceptor: Double = .0
     var iterations: Int = 0
     var weights: Array[Double] = null
+    var numMeans: Array[Double] = null
```

Comment:
True but it has to work with the underlying core H2O model for Model Outputs (that's also why I had to write parts of code in Java as the underlying framework requires several completely different constructors) - not sure if Option is an... option here :-) But will look into it.